components/Cards/index.js

```diff
@@ -17,3 +17,56 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+let cardData = axios.get('https://lambda-times-backend.herokuapp.com/articles');
```

Comment:
don't forget to do a .then() and .catch() with the axios call makes it easier to know if you got the data in the then or got an error in the catch